### ISSUE TITLE
Understand how the library works through flowcharts

### ISSUE THREAD

**Issue by gallegogt** [iota-community/IOTAStreams#7]
[Basic Public Channel Flow](https://app.diagrams.net/?lightbox=1&highlight=0000ff&edit=_blank&layers=1&nav=1&title=IotaStreamPoC.drawio#Uhttps%3A%2F%2Fdrive.google.com%2Fuc%3Fid%3D10IYiTxCGqzn2jLUO4QH21VQZHEAtAlqX%26export%3Ddownload)

[All Diagrams](https://drive.google.com/file/d/10IYiTxCGqzn2jLUO4QH21VQZHEAtAlqX/view?usp=sharing)

**Comment by JakeSCahill** [comment]
Hey @gallegogt, this is really helpful and should be included in the docs. Want to work on this together?

**Comment by gallegogt** [comment]
Hi @JakeSCahill, It would be great, I think that as more people contribute knowledge the documentation will be more complete.

**Comment by JakeSCahill** [comment]
Could you maybe add me as a contributor to your draw.io file?

We can discuss thing on Discord and make a plan to integrate it 😄

**Comment by JakeSCahill** [comment]
Could you maybe add me as a contributor to your draw.io file?

We can discuss on Discord and make a plan to integrate it 😄

**Comment by JakeSCahill** [comment]
Just to keep everyone up to date on this.

We are working on getting some example flowcharts for the following messaging workflows:

- Announcing the channel (a simplified version of this one)
- Publishing masked payloads
- Changing the author's signature keys

We've designed some icons for each message type and we're working with our graphic designer to IOTAify the flowcharts at the moment.

If anyone has any other suggestions related to these flowcharts or future ones, please add them here 😄 

As soon as we have an example, I'll share it here.

**Comment by JakeSCahill** [comment]
Hey everyone, 

What do you think of this style? The icons will be used throughout the docs to help you recognize message types.

![communication_channel](https://user-images.githubusercontent.com/45230295/81412478-d4f2c780-913b-11ea-8b33-c016d1b7a239.png)

**Comment by huhn511** [comment]
Could we also access these graphics for GitHub Repos/articles? It would be nice! :)

**Comment by JakeSCahill** [comment]
The PNGs will be on the documentation repository, do you mean the source SVGs?

**Comment by huhn511** [comment]
Yes, I mean the single boxes - so that we also can create some flows :)

**Comment by JakeSCahill** [comment]
Yeah of course. I'll check in with the designer

**Comment by JakeSCahill** [comment]
Single boxes are available here 😄  @huhn511 https://github.com/iotaledger/documentation/tree/develop/channels/1.0/images
